manifests/type/helm/template/storage.yaml

```diff
@@ -0,0 +1,89 @@
+ocfVersion: 0.0.1
+revision: 0.1.0
+kind: Type
+metadata:
+  name: storage
+  prefix: cap.type.helm.template
+  displayName: Helm template chart storage
+  description: Describes Helm storage
+  documentationURL: https://capact.io/docs
+  supportURL: https://helm.sh
+  maintainers:
+    - email: team-dev@capact.io
+      name: Capact Dev Team
+      url: https://capact.io
+
+spec:
+  additionalRefs:
+    - cap.core.type.hub.storage
+  jsonSchema:
+    value: |-
+      {
+        "$schema": "http://json-schema.org/draft-07/schema",
+        "type": "object",
+        "title": "The root schema",
+        "required": [
+          "url",
+          "contextSchema",
+          "acceptValue"
+        ],
+        "properties": {
+          "url": {
+            "$id": "#/properties/url",
+            "type": "string",
+            "format": "uri"
+          },
+          "contextSchema": {
+            "const": {
+              "$schema": "http://json-schema.org/draft-07/schema",
+              "type": "object",
+              "required": [
+                "goTemplate",
+                "release"
+              ],
+              "properties": {
+                "goTemplate": {
+                  "$id": "#/properties/context/properties/goTemplate",
+                  "type": "string"
+                },
+                "release": {
```

Comment:
probably we will need also a `chartLocaction` but I will add it in follow-up pr if necessary